glossary.md

```diff
@@ -87,6 +87,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | convolutional neural network        | mạng nơ-ron tích chập   |                                              |
 | corpus                              | kho ngữ liệu            |                                              |
 | correlation coefficient             | hệ số tương quan        |                                              |
+| cosine                              | cosin                   |                                              |
```

Comment:
![image](https://user-images.githubusercontent.com/30251584/86905801-cc6b3d00-c13c-11ea-8f52-71d72e1b23bb.png)
```suggestion
| cosine                              | côsin                   |                                              |
```

Reply:
Chỗ có dấu nối `–` này thấy từ vựng xưa họ hay dùng với từ mượn tiếng Anh, mà có lúc dùng có lúc không; như trong hình em Cường gửi thì lại không dùng. Không biết có nguyên tắc hay tài liệu gì để cập đến không nhỉ?

Mình thì có phần hay sử dụng dấu nối để dễ nhìn hơn, trong bảng thuật ngữ mình cũng đang có nhiều từ để dấu nối.

Reply:
Sách trên là sách giáo khoa à Cường?

Reply:
sgk toán 9 anh.

Reply:
Sách giáo khoa lớp 10 đây :-? 
![image](https://user-images.githubusercontent.com/30741785/86913795-1b739b00-c15a-11ea-8f5a-6b0fb186ac25.png)

Reply:
cái này là kí hiệu trong công thức toán rồi anh, mình đang cần cách viết trong văn xuôi cơ.

Reply:
à uh, a thấy có ảnh chính thức sgk 10 nên up vô đây để mn biết là côsin

Reply:
Thì bình thường trong công thức đều sử dụng viết tắt mà ông giáo 😅 nếu không nằm trong công thức thì mình vẫn prefer dùng đầy đủ hơn.